### DISCUSSION TITLE
Job to scan a volume for import new photos

### DISCUSSION THREAD

**Discussion by webysther** [immich-app/immich#1699] [Feature Request]
### The feature

Will be great to have a job in administration and a default mount location to run this job to automatic execute the import, example:

```sh
volume:
  - current_photos:/usr/src/app/import
```

This works great for photographers to use to import insane amount of photos without worries or without change the workflow.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by webysther** [comment]
Another sub feature maybe enable a few flags, like support create a albm from the last folder or using the same logic for creating the pattern to the current storage.

**Comment by bo0tzz** [comment]
The plan for this is to add a watch mode to the CLI. See https://github.com/immich-app/immich/issues/7 and https://github.com/immich-app/CLI/issues/18

**Reply by webysther** [reply]
Nice, thats it!

**Comment by bo0tzz** [comment]
Another nice way to do this would be to have an option to "absorb" an external library. That also gives you a preview of sorts before fully committing.
